GVFS/GVFS.Virtualization/Projection/Readme.md

```diff
@@ -0,0 +1,181 @@
+﻿# GitIndexProjection
+
+## Overview

```

Comment:
Perhaps it is worth mentioning that the `GVFS.Mount` process is keeping this projection in-memory throughout its lifetime.